src/Agent.Listener/Capabilities/AgentCapabilitiesProvider.cs

```diff
@@ -34,6 +35,10 @@ public Task<List<Capability>> GetCapabilitiesAsync(AgentSettings settings, Cance
 #if OS_WINDOWS
             Add(capabilities, "Agent.OSVersion", GetOSVersionString());
             Add(capabilities, "Cmd", Environment.GetEnvironmentVariable("comspec"));
+
+            //This capability gets set irrespective of agent being run in auto-logon mode or not.
+            var configurationManager = HostContext.GetService<IConfigurationManager>();
+            Add(capabilities, "InteractiveSession", (!configurationManager.IsServiceConfigured()).ToString());
```

Comment:
Pending on Chris/Prachi.

---
In reply to: [118593440](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r118593440) [](ancestors = 118593440)